R/theme.r

```diff
@@ -187,7 +187,14 @@ print.theme <- function(x, ...) utils::str(x)
 #'   plot.background  \tab background of the entire plot
 #'                    (\code{element_rect}; inherits from \code{rect}) \cr
 #'   plot.title       \tab plot title (text appearance)
-#'                    (\code{element_text}; inherits from \code{title}) \cr
+#'                    (\code{element_text}; inherits from \code{title})
+#'                    left-aligned by default\cr
+#'   plot.subtitle    \tab plot subtitle (text appearance)
+#'                    (\code{element_text}; inherits from \code{title})
+#'                    left-aligned by default\cr
+#'   plot.sub         \tab annotations below the plot (text appearance)
```

Comment:
s/plot.sub/plot.annotation/...

Want a PR against your repo or simple do the change in another commit?